tools/build_config/Jenkinsfile

```diff
@@ -190,6 +190,28 @@ pipeline {
       }
     }
 
+    stage('Configure') {
+      if (isUnix()) {
+        sh '''
+          export PATH=${PATH}:~/.local/bin &&
+          module load cmake/\$CMAKE_VERSION &&
+          module load matlab/\$MATLAB_VERSION &&
+          module load gcc/\$GCC_VERSION &&
+          module load cppcheck/\$CPPCHECK_VERSION &&
+          ./tools/build_config/build.sh \
+            --cmake_flags \"-DHorace_RELEASE_TYPE=\$RELEASE_TYPE\" \
+            --matlab_release \$MATLAB_VERSION \
+            --print_versions --configure
+          '''
+      } else {
+        powershell '''
+           ./tools/build_config/build.ps1 -print_versions -configure
+                -cmake_flags \"-DHorace_RELEASE_TYPE=\$env:RELEASE_TYPE\" \
+                -matlab_release \$env:MATLAB_VERSION \
+                -vs_version \$env:VS_VERSION
```

Comment:
Thanks for adding this flag. Can we add a ref to https://github.com/pace-neutrons/Horace/issues/593 in the PR description, so we know this is fixing the issue for Herbert?